### PR TITLE
Create a universal calibration file format

### PR DESCRIPTION
**Pull request by matthejo**
This change takes the existing JSON format used for camera intrinsics and extrinsics and combines them into a single file format, which can be serialized and deserialized using CalculatedCameraCalibration. I tested converting a locally-produced calibration file to this format and verified that it loads and works as expected. With this change there are no longer ICalibrationParsers. Instead CalibrationData is always used when receiving calibration files from the HMD.

### CODE REVIEW

File: Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/HolographicCamera/CalibrationDataProvider.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.IO;
+using System.Runtime.InteropServices.WindowsRuntime;
 using UnityEngine;
```
Comment (chrisfromwork) [inline Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/HolographicCamera/CalibrationDataProvider.cs]: Does WindowsRuntime resolve for non UNITY_WSA?

**Comment by chrisfromwork** [comment]
> using UnityEngine;

nit: i missed this before, please add a license stub to this file

---
Refers to: Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/CalibrationData.cs:2 in dbae0c9. [](commit_id = dbae0c9a6a176e9d24750b718c718122dd926fb0, deletion_comment = False)

**Comment by chrisfromwork** [comment]
> using System;

nit: i missed this before, please add a license stub to this file.

---
Refers to: Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/HolographicCamera/CalibrationDataProvider.cs:2 in dbae0c9. [](commit_id = dbae0c9a6a176e9d24750b718c718122dd926fb0, deletion_comment = False)

File: Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/HolographicCamera/CalibrationDataProvider.cs (hunk 1 above)
Comment (provencher) [reply Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/HolographicCamera/CalibrationDataProvider.cs]: I had issues with some Interop service APIs not working on IL2CPP with unity wsa too. I'm a bit weary

Comment (matthejo) [reply Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/HolographicCamera/CalibrationDataProvider.cs]: This particular method seems to be working on IL2CPP (x86). I've moved the using into the UNITY_WSA-specific block.

**Pull request merged**